### PR TITLE
[ADD] l10n_mu_account: Mauritius accounting package

### PR DESCRIPTION
**Pull request by clbr-odoo**
Adds Mauritius localization, includes:
    - Chart of accounts
    - Taxes
    - Fiscal positions
    - Default settings, including invoice customization.

task-3631730

### CODE REVIEW

File: addons/l10n_mu_account/data/tax_report-mu.xml [hunk 1]
```diff
@@ -0,0 +1,472 @@
...
+                            <record id="mu_tr_T12_B_tag" model="account.report.expression">
+                                <field name="label">tag</field>
+                                <field name="engine">tax_tags</field>
+                                <field name="formula">T12_B</field>
+                            </record>
+                            <record id="mu_tr_T12_B_vat" model="account.report.expression">
+                                <field name="label">vat</field>
+                                <field name="engine">aggregation</field>
+                                <field name="formula">T12.tag + T12._applied_carryover_balance</field>
+                            </record>
```
Comment (epictete) [inline addons/l10n_mu_account/data/tax_report-mu.xml:332]:
```suggestion
                            <record id="mu_tr_T12_applied_carryover" model="account.report.expression">
                                <field name="label">_applied_carryover_balance</field>
                                <field name="engine">external</field>
                                <field name="formula">most_recent</field>
                                <field name="date_scope">previous_tax_period</field>
                            </record>
                            <record id="mu_tr_T12_tag" model="account.report.expression">
                                <field name="label">tag</field>
                                <field name="engine">tax_tags</field>
                                <field name="formula">T12_tag</field>
                            </record>
                            <record id="mu_tr_T12_vat" model="account.report.expression">
                                <field name="label">vat</field>
                                <field name="engine">aggregation</field>
                                <field name="formula">T12.tag + T12._applied_carryover_balance</field>
                            </record>
```
(If you agree with the previous comment :smile:)

**Comment by clbr-odoo** [comment]
Thank you for your review @epictete ! 
I agree with your comments and updated accordingly :+1:

**Review by epictete — APPROVED** [review]
Thanks for he quick update @clbr-odoo :ok_hand: 

@robodoo r+

**Comment by epictete** [comment]
@robodoo r+

**Pull request closed without merging**